### PR TITLE
Make Isoline respect linewidths and line attributes

### PR DESCRIPTION
**Pull request by chaosphere2112**
I started work on this fix for #1625, but it quickly got out of my VTK depth. @sankhesh, I stubbed in the logic for applying the width and patterns (and I built the patterns for you, so you don't have to futz with those). Mind making this work properly and apply only to specific levels?

I was planning on doing a per-level vtkContourFilter > vtkPolyDataMapper > vtkActor and setting the property appropriately for each of those, but if there's a better way then by all means, do so.

### CODE REVIEW

**Comment by sankhesh** [comment]
:ok: @chaosphere2112 I'll take a look.

**Comment by aashish24** [comment]
@sankhesh ping!

**Comment by chaosphere2112** [comment]
@sankhesh FYI, just found the patterns that were already defined elsewhere in the codebase:

```
        if t == 'long-dash':
            p.SetLineStipplePattern(int('1111111100000000', 2))
            p.SetLineStippleRepeatFactor(1)
        elif t == 'dot':
            p.SetLineStipplePattern(int('1010101010101010', 2))
            p.SetLineStippleRepeatFactor(1)
        elif t == 'dash':
            p.SetLineStipplePattern(int('1111000011110000', 2))
            p.SetLineStippleRepeatFactor(1)
        elif t == 'dash-dot':
            p.SetLineStipplePattern(int('0011110000110011', 2))
            p.SetLineStippleRepeatFactor(1)
        elif t == 'solid':
            p.SetLineStipplePattern(int('1111111111111111', 2))
            p.SetLineStippleRepeatFactor(1)
        else:
            raise Exception("Unknown line type: '%s'" % t)
```

**Comment by aashish24** [comment]
@sankhesh are you planning to look at it soon?

File: Packages/vcs/Lib/vcsvtk/isolinepipeline.py [hunk 1]
```diff
@@ -6,6 +6,15 @@
 import vtk
 
 
+LINE_PATTERNS = {
```
Comment (aashish24) [inline Packages/vcs/Lib/vcsvtk/isolinepipeline.py]: Do we need it at this scope? May be we can define it inside the class itself?

**Comment by sankhesh** [comment]
@chaosphere2112 @aashish24 
I haven't forgotten about this. I am looking at #1086 which is a critical issue. I'll look at this soon.

**Comment by sankhesh** [comment]
@chaosphere2112 I had to abandon this branch and push a different branch as I had a lot of changes.
Please refer to #1688

**Pull request closed without merging**